.github/actions/spelling/excludes.txt

```diff
@@ -90,6 +92,7 @@
 ^pkg/rancher-desktop/assets/scripts/logrotate-k3s$
 ^pkg/rancher-desktop/assets/scripts/logrotate-lima-guestagent$
 ^pkg/rancher-desktop/sudo-prompt/
+^pkg/rancher-desktop/utils/processOutputInterpreters/__tests__/assets/trivy-image-postgres
```

Comment:
There are a pair of files that had a bunch of things that check-spelling doesn't like, but they're clearly designed as inputs to a test based on real world values, so fixing them doesn't make sense.